DeskKit/DeskKit/View Models/DKContactUsViewModel.m

```diff
@@ -99,6 +103,22 @@ - (NSArray *)sections
     return _sections;
 }
 
+- (NSAttributedString *)attributedTextWithString:(NSString *)string
+{
+    if (string) {
+        return [[NSAttributedString alloc] initWithString:string];
+    }
+    return nil;
+}
+
+- (NSAttributedString *)attributedPlaceholderTextWithString:(NSString *)string
+{
+    if (string) {
+        return [[NSAttributedString alloc] initWithString:string];
+    }
+    return nil;
+}
```

Comment:
In that case, I'd probably leave them in there like you have them or just have `attributedPlaceholderTextWithString` invoke `attributedTextWithString`.